Compilation of ngeo examples fails with new CC version

@fredj merged an OpenLayers PR to use the newest versions of Closure Compiler and Closure Library. I just run `rm -rf node_modules/openlayers && npm install && make check`  in ngeo and I get errors when compiling the examples code:

```
node buildtools/build.js buildtools/examples-all.json .build/examples/all.min.js
info ol Parsing dependencies
info ol Compiling 371 sources
ERR! compile .build/examples/all.js:27: ERROR - Closure dependency methods(goog.provide, goog.require, etc) must be called at file scope.
ERR! compile goog.provide('animation');
ERR! compile ^
ERR! compile 
ERR! compile .build/examples/all.js:92: ERROR - Closure dependency methods(goog.provide, goog.require, etc) must be called at file scope.
ERR! compile goog.provide('control');
ERR! compile ^
ERR! compile 
ERR! compile 
ERR! compile .build/examples/all.js:142: ERROR - Closure dependency methods(goog.provide, goog.require, etc) must be called at file scope.
ERR! compile goog.provide('geolocation');
ERR! compile ^
ERR! compile 
ERR! compile .build/examples/all.js:209: ERROR - Closure dependency methods(goog.provide, goog.require, etc) must be called at file scope.
ERR! compile goog.provide('interactionbtngroup');
ERR! compile ^
ERR! compile 
ERR! compile .build/examples/all.js:305: ERROR - Closure dependency methods(goog.provide, goog.require, etc) must be called at file scope.
ERR! compile goog.provide('interactiontoggle');
ERR! compile ^
ERR! compile 
ERR! compile .build/examples/all.js:366: ERROR - Closure dependency methods(goog.provide, goog.require, etc) must be called at file scope.
ERR! compile goog.provide('layercatalog');
ERR! compile ^
ERR! compile 
ERR! compile .build/examples/all.js:524: ERROR - Closure dependency methods(goog.provide, goog.require, etc) must be called at file scope.
ERR! compile goog.provide('layermanager');
ERR! compile ^
ERR! compile 
ERR! compile .build/examples/all.js:662: ERROR - Closure dependency methods(goog.provide, goog.require, etc) must be called at file scope.
ERR! compile goog.provide('layeropacity');
ERR! compile ^
ERR! compile 
ERR! compile .build/examples/all.js:703: ERROR - Closure dependency methods(goog.provide, goog.require, etc) must be called at file scope.
ERR! compile goog.provide('layervisibility');
ERR! compile ^
ERR! compile 
ERR! compile .build/examples/all.js:755: ERROR - Closure dependency methods(goog.provide, goog.require, etc) must be called at file scope.
ERR! compile goog.provide('permalink');
ERR! compile ^
ERR! compile 
ERR! compile .build/examples/all.js:862: ERROR - Closure dependency methods(goog.provide, goog.require, etc) must be called at file scope.
ERR! compile goog.provide('simple');
ERR! compile ^
ERR! compile 
ERR! compile 11 error(s), 0 warning(s)
ERR! compile 
ERR! Process exited with non-zero status, see log for more detail: 11 
make: *** [.build/examples/all.min.js] Error 1
```

--- a/buildtools/combine-examples.py
+++ b/buildtools/combine-examples.py
@@ -12,7 +12,9 @@ def main(argv):
         if len(lines) > 0 and lines[0].startswith('// NOCOMPILE'):
             continue
         requires.update(line for line in lines if line.startswith('goog.require'))
-        examples[filename] = [line for line in lines if not line.startswith('goog.require')]
+        examples[filename] = [line for line in lines if \
+            not (line.startswith('goog.require') or \
+                 line.startswith('goog.provide'))]
     for require in sorted(requires):
         sys.stdout.write(require)
     for filename in sorted(examples.keys()):

